aws/modules/consolidate-logs/module.ftl

```diff
@@ -0,0 +1,226 @@
+                                        "Version" : "",
+                                        "Function" : "processor",
+                                        "Role" : "invoke"
+                                    }
+                                }
+                            }
+                        },
+                        lambdaName : {
+                            "lambda": {
+                                "Instances": {
+                                    "default": {
+                                        "DeploymentUnits": [ lambdaName ]
+                                    }
+                                },
+                                "Image" : {
+                                    "Source" : "url",
+                                    "UrlSource" : {
+                                        "Url" : lambdaSourceUrl,
+                                        "ImageHash" : lambdaSourceHash
+                                    }
+                                },
+                                "Functions": {
+                                    "processor": {
+                                        "RunTime": "python3.6",
+                                        "MemorySize": 128,
+                                        "Timeout": 30,
+                                        "Handler": "src/run.lambda_handler",
+                                        "Links": {
+                                            "store": {
+                                                "Tier": "mgmt",
+                                                "Component": "baseline",
+                                                "DataBucket" : "opsdata",
+                                                "Instance" : "",
+                                                "Version": "",
+                                                "Role": "datafeed"
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "LogFilters": {
+                "all-logs": {
+                    "Pattern": ""
+                }
+            },
+            "LoggingProfiles" : {
+                "consolidate" : {
+                    "ForwardingRules": {
+                        "store": {
+                            "Filter": "all-logs",
+                            "Links": {
+                                "store": {
+                                    "Tier": "mgmt",
+                                    "Component": "baseline",
+                                    "Instance": "",
+                                    "Version": "",
+                                    "DataBucket" : "opsdata",
+                                    "Role" : "produce"
+                                }
```

Comment:
I think that link should be to the datafeed rather than the opsdata bucket

Reply:
Yep, frustratingly I had this right the first time around and I replicated it poorly into the module